Xamarin.Essentials/Browser/Browser.shared.cs

```diff
@@ -37,7 +37,7 @@ internal static Uri EscapeUri(Uri uri)
 
     public enum BrowserLaunchMode
     {
+        SystemPreferred = 0,
         External,
```

Comment:
Should we put = 1 here?

Reply:
I have no real strong opinion here, but maybe for consistency?

Reply:
Not against putting the 1 as well but I do like having 0 so it's obvious where it starts...

I think having 0 is sufficient. But again not against adding 1,2 etc to the others